client/src/lib/components/CaptchaV2.svelte

```diff
@@ -1,9 +1,10 @@
-<script lang="ts">
+<script type="module" lang="ts">
   import { createEventDispatcher, onMount } from "svelte";
   import Cross from "./icons/Cross.svelte";
+  import { CaptchaProvider } from "$lib/utils/captcha";
 
   export let captchaKey: string;
-
+  export let captchaProvider: string;
```

Comment:
Just a tiny change here for type safety:
```suggestion
  export let captchaProvider: CaptchaProvider;
```

Reply:
Done.